torch/_dynamo/skipfiles.py

```diff
@@ -164,37 +161,23 @@ def _module_dir(m: types.ModuleType):
     "torch._constrain_as_value",
 }
 
-
-# Force inline functions in these files or directories, even they are in *_SKIPLIST.
-# We are using python module name instead of file or directory object to avoid circular dependency.
-# Please keep this sorted alphabetically.
-# TODO: Merge FILE_INLINELIST into SUBMODULE_INLINELIST.
-FILE_INLINELIST = {
-    "torch._dynamo._trace_wrapped_higher_order_op",
-    "torch._dynamo.comptime",
+LEGACY_MOD_INLINELIST = {
```

Comment:
docs? So I guess the idea is we are one-by-one going to move these to MOD_INLINELIST and fix problems that occur?

Reply:
I think those left in the ```LEGACY_MOD_INLINELIST``` are legacy workaround. I still remember those conversation (with export and ao team folks) when adding these workaround. Ideally we should remove these. But the basic principle is, DO NOT add new one to the legacy list.